content/docs/2.3/scalers/selenium-grid-scaler.md

```diff
@@ -0,0 +1,75 @@
++++
+title = "Selenium Grid Scaler"
+availability = "v2.3+"
+maintainer = "Volvo Cars"
+description = "Scales selenium browser nodes based on number of requests waiting in session queue"
+layout = "scaler"
+go_file = "selenium_grid_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `selenium-grid` trigger that scales browser nodes based on number of requests in session queue.
+
+The scaler creates one browser node per pending request in session queue. You will have to create one trigger per browser capability that you would like to support in your selenium grid.
+
+The below is an example trigger configuration for chrome node.
+
+```yaml
+triggers:
+  - type: selenium-grid
+    metadata:
+      url: 'http://selenium-hub:4444/graphql'
+      browserName: 'chrome'
+```
+
+**Parameter list:**
+
+- `url` is graphql url of your selenium grid. Refer to the selenium grid's documentation [here](https://www.selenium.dev/documentation/en/grid/grid_4/graphql_support/) to for more info.
+- `browserName` is the name of browser that usually gets passed in the browser capability.
```

Comment:
Minor nit, is it possible to list the allowed values in here? Or add a link to the relevant selenium documentation>

Reply:
Added link to the documentation